src/param.rs

```diff
@@ -272,9 +274,11 @@ pub(crate) struct AudioParamRaw {
     default_value: f32, // immutable
     min_value: f32,     // immutable
     max_value: f32,     // immutable
+    automation_rate: AutomationRate,
```

Comment:
Including the non-Arc field here breaks the `Clone` implementation (which we need for the node bindings).
Calling set_automation_rate on a cloned AudioParam will not update the automation rate of the original AudioParam